ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -56,6 +56,7 @@ while true; do
   sleep 10
 done
 
+make keystone_kuttl
```

Comment:
we need to split operations here, the deploy step already deploy things, can't we just run tests? if yes, can we add them to a new step?

Reply:
Hi @viroel From what I understand, our current KUTTL tests are written to run against individual operator CRs and not with Meta CRs. @cescgina and I were discussing this, and we think that if we want to run KUTTL tests after deploying with the openstack operator, we will need to make some changes to the tests.

I wanted to get your thoughts on this - do you think it's okay for us to run the KUTTL tests for the service operator under test and then cleanup everything we installed with KUTTL before deploying the openstack operator, at least for the time being? If we decide to modify the tests so that they can be run after deploying with the openstack operator, we can switch to just running KUTTL tests after deployment with the meta operator. Let me know what you think!

Reply:
Since kuttl tests are written to run against a specific deployment, I think that a new kuttle step needs to be created and this new step should also deploy operators that kuttl expects (which may be the minimal deployment to have the operator working?). After running kuttl tests, it would be good to cleanup the environment too (if possible).
I don't think we should mix things here, or It might get complicated to maintain a nsingle deploy step.
So yes, I think that we agree on that subject :)
It would be good to have a minimal deployment for each operator too (without meta)

Reply:
@viroel thanks for your input.

I agree with you about creating a separate step for running the kuttl tests.

Actually, We already created two targets in install_yamls for running kuttl tests:
1) For just running the kuttl tests(This assumes we have everything we need to run kuttl tests)
2) This installs the operators before running kuttl tests and also does the cleanup after running kuttl. (We are going to use this target in CI) example[1]

[1] 
https://github.com/openstack-k8s-operators/install_yamls/blob/master/Makefile#L852-L857

~~~
.PHONY: keystone_kuttl
keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy mariadb_deploy_validate keystone_deploy_prep keystone ## runs kuttl tests for the keystone operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
	make keystone_kuttl_run
	make deploy_cleanup
	make keystone_cleanup
	make mariadb_cleanup
~~~